languages/cpp/ExampleUse.md

```diff
@@ -48,23 +55,68 @@ Example of the folder structure (macOS):
         --`schemas.hpp`
     --examples
         --`Wrapper.cpp`
+```
+
+Set the environment variable path for the Bitwarden libraries.
+
+For macOS:
+
+```bash
+export DYLD_LIBRARY_PATH=/path/to/your/library:$DYLD_LIBRARY_PATH
+```
+
+For Linux:
+
+```bash
+export LD_LIBRARY_PATH=/path/to/your/library:$LD_LIBRARY_PATH
+```
 
+For Windows:
 
-1. $ export ACCESS_TOKEN=<"access-token">
-2. $ export ORGANIZATION_ID=<"organization-id">
-3. $ export DYLD_LIBRARY_PATH=/path/to/your/library:$DYLD_LIBRARY_PATH
+```shell
+ set PATH=%PATH%;C:\path\to\your\library
+```
 
-The last step is neccessary to add the path for the dynamic library (macOS).
-For the Linux one should use:
-$ export LD_LIBRARY_PATH=/path/to/your/library:$LD_LIBRARY_PATH
-For the Windows:
-$ set PATH=%PATH%;C:\path\to\your\library
+Set environment variables used in `Wrapper.cpp`:
 
-4. $ cd examples
-5. $ clang++ -std=c++20 -I../include -I/path/to/include/nlohmann  -I/path/to/include/boost -L../build/ -o MyBitwardenApp Wrapper.cpp -lBitwardenClient -ldl
+```bash
+export ACCESS_TOKEN=<"access-token">
+export ORGANIZATION_ID=<"organization-id">
+export API_URL=http://localhost:4000
+export IDENTITY_URL=http://localhost:33656
+```
+
+Compile:
+
+```bash
+cd examples
+clang++ -std=c++20 -I../include -I/path/to/include/nlohmann  -I/path/to/include/boost -L../build/ -o MyBitwardenApp Wrapper.cpp -lBitwardenClient -ldl
+```
 
 for Windows `-ldl` should be excluded,
 
-The result is `MyBitwardenApp` in the `examples` directory, and one can run it from the `examples` directory:
+The result is `MyBitwardenApp` in the `examples` directory, and can be ran from the `examples` directory:
+
+```bash
+./MyBitwardenApp
+```
+
+## Example
+
+### macOS
```

Comment:
Shouldn't this be part of `### Compiling` ?

Reply:
Yes, it should updated in https://github.com/bitwarden/sdk/pull/854/commits/eec019b5c8623ee1902f53c7804e131c0b405445